apps/wallet/src/shared/cryptography/keystore.ts

```diff
@@ -2,16 +2,24 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import passworder from '@metamask/browser-passworder';
+
+type Serializable =
+    | string
+    | number
+    | boolean
+    | Record<string, string | number | boolean>
+    | { [index: string]: Serializable };
```

Comment:
This index type and the `Record` type above are basically the same thing.

Reply:
Thanks! `index` was suppose to be an improvement of using the `Record` and I forgot to remove it